src/scripts/main.js

```diff
@@ -1,5 +1,43 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  const spider = document.querySelector('.spider');
+  const wall = document.querySelector('.wall');
+  const widthSpider = spider.offsetWidth;
+  const borderWidth = Array.from(getComputedStyle(wall).borderTopWidth);
+  const borderWall = borderWidth
+    .filter((x) => x / x === 1 || x === '.')
+    .join('');
+
+  wall.style.cssText = 'relative';
```

Comment:
It seems like you're trying to set the position property of the wall to 'relative', but the syntax is incorrect. It should be `wall.style.position = 'relative';`.